CHANGELOG.rst

```diff
@@ -45,6 +45,9 @@ Fixed
 - Calls to setExternalCharges were failing if number of external charges
   changes.
 
+- Incorrect excited gradients for spin-polarized, long-range corrected
```

Comment:
```suggestion
- Incorrect excited gradients for spin-polarized long-range corrected
```
As could be read that spin-polarised are generally affected, instead of specific to LR

Reply:
@bhourahine applied